.pre-commit-search-and-replace.yaml

```diff
@@ -35,3 +35,7 @@
 - search: ', optional, default:'
   replacement: ', default:'
   description: No need to specify that a parameter is optional if there is a default value
+
+- search: 'the the '
```

Comment:
@namurphy: Not 100% sure, if this matches on something, that should not be touched.

If ' the the' is used, without trailing space, things like "the theme, the theory" are matched. But if there should be a leading space, I am not sure. There might also be a comma.

What do you think?

Reply:
This will catch most of the `the the`'s with very few false positives.  We could potentially create a regular expression that makes use of match groups if we wanted to be more precise, but I don't think it'd be worth the effort.

Thank you again!

Reply:
Excellent, glad I could help out.